### ISSUE TITLE
js closureCompiler.run() signature is wrong in README

### ISSUE THREAD

**Issue by leeoniya** [google/closure-compiler-npm#94]
hey @ChadKillingsworth,

```js
const compilerProcess = closureCompiler.run(function(exitCode, stdOut, stdErr) {
  //compilation complete
}, [{
 path: 'file-one.js',
 src: 'alert("hello world")',
 sourceMap: null // optional input source map
}]);
```

this should be `run(fileList, callback)` [1] ... and the return value is the compilation artifact `out`.

i can confirm that https://github.com/google/closure-compiler-js/issues/79 is resolved, btw.

[1] https://github.com/google/closure-compiler-npm/blob/master/lib/node/closure-compiler-js.js#L52

**Comment by ChadKillingsworth** [comment]
Oh nice catch. I'll correct that.

**Comment by ChadKillingsworth** [comment]
Fixed by #98
